app/controllers/events_controller.rb

```diff
@@ -0,0 +1,44 @@
+class EventsController < ApplicationController
+  include SessionsHelper
+  def new
+    redirect_to root_path if current_user.nil?
+    @event = Event.new
+  end
+
+  def index
+    @past_events = Event.where('event_date < ?', DateTime.now)
+    @present_events = Event.where('event_date >= ?', DateTime.now)
```

Comment:
**[REQUIRED]**
Let's use scope in the Event model so that we can just access the past event as follows
```
@past_events = Event.previous_events
```
In the event model we can have 
```
scope :previous, -> { where('date < ?', Time.zone.now) }
```
Let's do the same for the upcoming events
([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/scopes_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))
([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/past_upcoming_events_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))